react/components/__tests__/QuantitySelector.test.tsx

```diff
@@ -0,0 +1,240 @@
+import React from 'react'
+import { render, screen, fireEvent } from '@vtex/test-tools/react'
+import { ToastContext } from '@vtex/styleguide/lib/ToastProvider'
+
+import QuantitySelector from '../QuantitySelector'
+
+describe('<QuantitySelector />', () => {
+  it('should change from dropdown to input after quantity exceeds max value', () => {
+    const onChange = jest.fn()
+
+    render(
+      <QuantitySelector id="1" value={5} maxValue={50} onChange={onChange} />
+    )
+
+    const [dropdown] = screen.getAllByRole('combobox')
```

Comment:
Why not:
```suggestion
    const dropdown = screen.getByRole('combobox')
```

Reply:
Just saw this bellow

> expect(dropdown).toBeInTheDocument()

So, the point is in case it fails to get the dropdown, the error message will be better?

Reply:
I can't use `getByRole` because we have more than one dropdown in the page